.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
+.env
+/node_modules/
```

Comment:
please add also `.vs` and `.vscode`  to gitignore

Reply:
and also remove the vs nd vscode files from the commit

Reply:
okay done